Assets/MixedRealityToolkit-SDK/Features/Boundary/System/MixedRealityBoundarySystem.cs

```diff
@@ -38,14 +38,22 @@ private void InitializeInternal()
         {
             boundaryEventData = new BoundaryEventData(EventSystem.current);
 
-            Scale = MixedRealityManager.Instance.ActiveProfile.TargetExperienceScale;
+            boundaryVisualizationParent = new GameObject("Boundary System");
```

Comment:
`Boundary System Visualization`?

Reply:
I think leaving off the last part is a bit misleading, and might have devs looking for a boundary system component on it, when most likely none will exist.